readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -273,22 +266,34 @@ def build(self):
             *self.sphinx_parallel_arg(),
         ]
         if self.config.sphinx.fail_on_warning:
-            build_command.extend(['-W', '--keep-going'])
-        build_command.extend([
-            '-b',
-            self.sphinx_builder,
-            '-d',
-            '_build/doctrees',
-            '-D',
-            'language={lang}'.format(lang=project.language),
-            '.',
-            self.sphinx_build_dir,
-        ])
+            build_command.extend(["-W", "--keep-going"])
+        build_command.extend(
+            [
+                "-b",
+                self.sphinx_builder,
+                "-d",
+                self.sphinx_doctrees_dir,
+                "-D",
+                f"language={project.language}",
+                # Execute the command at the clone directory,
+                # but use `docs/` instead of `.` for the Sphinx path
+                os.path.dirname(
+                    os.path.relpath(
+                        self.config_file,
+                        self.project_path,
+                    ),
+                ),
+                self.sphinx_build_dir,
+            ]
+        )
         cmd_ret = self.run(
             *build_command,
-            cwd=os.path.dirname(self.config_file),
             bin_path=self.python_env.venv_bin(),
+            cwd=self.project_path,
         )
```

Comment:
Seems like this is where we can remove this change in `cwd`? I'm 👍 on undoing this change.

Reply:
I revert this change to use the CWD we were using previously.